.github/workflows/build-portal-client-docker-cache.yml

```diff
@@ -0,0 +1,31 @@
+name: Portal Client Docker Build and Save Cache
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '**.md**'
+    branches: [main]
+
+jobs:
+  build-and-cache-portal-client:
+      uses: USSF-ORBIT/ussf-portal/.github/workflows/reusable-docker-cache-portal-client.yml@main
+
+  trigger-build-and-cache-in-other-repos:
+      runs-on: ubuntu-latest
+      steps:
+        - name: call GH API to trigger caching workflow in CMS repo
+          run: gh workflow run build-portal-client-docker-cache.yml -R USSF-ORBIT/ussf-portal-cms --ref main
+          env:
+            GH_TOKEN: ${{secrets.PUBLIC_REPO_READ}} 
```

Comment:
This Personal Access Token is from the `spacecadets-infra@truss.works` GH account and only has public repo read access.